mycore-solr/src/main/java/org/mycore/solr/commands/MCRSolrCommands.java

```diff
@@ -131,7 +131,8 @@ public static void switchSolrCore(String coreID1, String coreID2) {
             + "by using the Solr schema api for core with the id {0}",
         order = 70)
     public static void reloadSolrConfiguration(String configType, String coreID) {
-        MCRSolrSchemaReloader.clearSchema(configType, coreID);
+        MCRSolrConfigReloader.cleanConfig(configType, coreID);
+        MCRSolrSchemaReloader.cleanSchema(configType, coreID);
```

Comment:
Shouldn't this stay clear?
Like List.clear() or clearing a cache? You don't wipe it clean, but remove something. You are clearing it up.